### DISCUSSION TITLE
Reposition "stoplight" buttons on Mac?

### DISCUSSION THREAD

**Discussion by iansinnott** [wailsapp/wails#2210] [Q&A]
Most native Mac windows include more padding around stoplight buttons than the default. The screenshot below is of Finder.

![CleanShot 2022-12-15 at 15 58 12@2x](https://user-images.githubusercontent.com/3154865/207805398-87542130-6d25-46df-a4b2-fab390d61aaa.jpg)

This "more padding" look seems to be the new design aesthetic apple is pushing, and it would be great if wails apps could look exactly like native apps.

I tried using `Frameless: true` which would allow re-creating the stoplight buttons with CSS, however, frameless mode removes the rounded corners on the window.

**Comment by leaanthony** [comment]
Wails uses the standard Cocoa APIs to create windows so I'm surprised there's inconsistencies. Do you know if there are new options for NSWindows in newer macOS versions? If you can find it, we can do it!
